limits/resources/redis/lua_scripts/acquire_moving_window.lua

```diff
@@ -17,7 +17,10 @@ local entries= {}
 for i=1, amount do
     entries[i] = timestamp
 end
-redis.call('lpush', KEYS[1], unpack(entries))
+
+for i, entry in ipairs(entries) do
```

Comment:
Since `entries` was previously only being populated so that we could do a single `lpush` call - we might as well move the call into the above loop that was populating `entries` and get rid of it entirely.

Reply:
You are correct. Removed it